content/docs/conditional-rendering.md

```diff
@@ -238,6 +238,18 @@ ReactDOM.render(
 );
 ```
 
+In this refactored version of `<WarningBanner />` (below), you can see the benefits of returning `false` over `null`. Here we use a logical AND to return either `false` or the rendered warning.
```

Comment:
Would rather explain the benefit than leave it as an exercise to the reader. I assume you are saying the `&&` is shorter and inline-able?

Reply:
Any better?
> In this refactored version of `<WarningBanner />` (below), the benefits of returning `false` over `null` is that `false` is the natural result of `props.warn` – i.e. no additional values need to be returned after the condition is determined. Here we are using a logical AND to return either `false` or the rendered warning, or to put it in plain English: "return `false` if the `warn` prop is `false`, otherwise render the warning".

I'd also like to link to [this article](https://medium.freecodecamp.org/conditional-rendering-in-react-using-ternaries-and-logical-and-7807f53b6935) to explain it more, but not sure about Facebook's policy on this.